pom.xml

```diff
@@ -126,6 +131,16 @@
                   <exclude>webapp</exclude>
                 </excludes>
               </generator>
+              <enricher>
+                <includes>
+                  <include>wildfly-swarm-health-check</include>
+                </includes>
+                <config>
+                  <wildfly-swarm-health-check>
+                    <path>/</path>
```

Comment:
Of course "/" returns index.html which is only available when application is ready.